package.json

```diff
@@ -32,6 +35,7 @@
     "build:extras": "bash minify-extras.sh",
     "min": "npm run min:systemjs && npm run min:sjs",
     "min:systemjs": "cd dist && terser system.js -c passes=2 -m --source-map --comments \"/SystemJS \\d/\" -o system.min.js",
+    "min:systemjs-node": "cd dist && terser system-node.js -c passes=2 -m --source-map --comments \"/SystemJS \\d/\" -o system-node.min.js",
```

Comment:
I'm not sure this is need since we don't need to minimize the code for node. I'll remove it.